docs/docs/thinking-in-react.md

```diff
@@ -113,7 +113,7 @@ Remember: React is all about one-way data flow down the component hierarchy. It
 For each piece of state in your application:
 
   * Identify every component that renders something based on that state.
-  * Find a common owner component (a single component above all the components that need the state in the hierarchy).
+  * Find a common owner component (a single component above all the other components that needs the state in the hierarchy).
```

Comment:
It's subtle but I think this was more correct as written. The other components need the data in the hierarchy. The single component above doesn't.